device.go

```diff
@@ -12,185 +12,5 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/*
-Package awsiotdev implements offline queueing and reconnecting features of MQTT protocol.
-*/
+// Package awsiotdev implements offline queueing and reconnecting features of MQTT protocol.
 package awsiotdev
```

Comment:
Do we still need this file?

Reply:
This is just for GoDoc. `awsiotdev.go` may be better filename.

Reply:
updated the filename